pkgs/by-name/nf/nfs-utils/package.nix

```diff
@@ -109,16 +113,12 @@ stdenv.mkDerivation rec {
       sha256 = "1fqws9dz8n1d9a418c54r11y3w330qgy2652dpwcy96cm44sqyhf";
     })
     (fetchpatch {
-      url = "https://raw.githubusercontent.com/void-linux/void-packages/bb636cdb1b274f44d92b1cb2fdf0dff6079f97aa/srcpkgs/nfs-utils/patches/nfs-utils-2.7.1-define_macros_for_musl.patch";
-      hash = "sha256-wsyioRjzs1PObMHwYgf5h/Ngv+s5MPsroAuUNGs9lR0=";
+      url = "https://github.com/void-linux/void-packages/raw/31f0d5fef2f74999212bcfa6f982969973432750/srcpkgs/nfs-utils/patches/musl-includes.patch";
```

Comment:
this redirects to raw.githubusercontent - do we have a clear policy on which to prefer?

Reply:
I don't think there's a clear policy. I can use raw.githubusercontent.com if you prefer.

Reply:
no strong preference